kore/src/Kore/Attribute/Overload.hs

```diff
@@ -87,3 +94,24 @@ instance ParseAttributes Overload where
       where
         toAttribute (symbol1, symbol2) =
             Attributes [overloadAttribute symbol1 symbol2]
+
+instance ParseAttributes (Overload Symbol) where
+    parseAttribute _                    (Overload Nothing)           =
+        return $ Overload Nothing
+
+    parseAttribute attrPattern overload@(Overload (Just symbolPair)) = do
```

Comment:
This is the part I'm a little concerned about, especially because the unit tests passed even when I made the result `undefined`

Reply:
This instance should not exist. It is not possible to parse an `Overload Symbol`. There are fields of `Symbol` missing, and I have no idea why that isn't an error.